CONTRIBUTING.md

```diff
@@ -1,14 +1,109 @@
+
+**Markdown Page Files**: 
+- These files render to html and make up the individual pages of the site. Most of the site's documentation text content is written in `md` files.
+- Change these to make changes to individual pages' content/text or markup. 
+- Each translation has it's own complete set of pages, located under their respective language directories. Ex. English middleware page is located at `en > resources > middleware.md.`
+
+**Template Includes and Layout Files**
+- These file are page components that make up the user interface and periphery structure. Ex. Header, Footer, etc.
+- There are also markdown files here that are *included* within other larger files. Ex `api > en` holds the API Reference text content.
+- Change these to make changes to page layouts or site-wide structures, or to change the API Reference documentation.
+- Located mainly under `_includes` and `_layouts`. API markdown and text content are located under `_includes/api`.
+
+**Blog Markdown Files**
+- These files make up the individual blog posts. If you want to contribute a blog post please
+follow the specific instructions for [How to write a blog post.](https://expressjs.com/en/blog/write-post.html)
+- Located under the `_posts` directory. 
+
+**CSS or Javascript**
+- All css and js files are kept in `css` and `js` folders on the project root.
+
+#### Step3: Running the Application
+
+
+Now you'll need a way to see your changes, which means you'll need a running version of the application. You have two options. 
+1. __Run Locally__: This gets the local version of the application up and running on your machine. Follow our [Local Setup Guide](https://github.com/expressjs/expressjs.com?tab=readme-ov-file#local-setup) to use this option.  
+    - This is the recommended option for moderate to complex work. 
+2. __Run using Deploy Preview__: Use this option if you don't want to bother with a local installation. Part of our continuous integration pipeline includes [Netlify Deploy Preview](https://docs.netlify.com/site-deploys/deploy-previews/). 
+    - To use this you'll need to get your changes online - after you've made your first commit on your feature branch, make a *draft* pull request. 
+    - After the build steps are complete, you'll have access to a __Deploy Preview__ tab that will run your changes on the web, rebuilding after each commit is pushed. 
+    - After you are completely done your work and it's ready for review, remove the draft status on your pull request and submit your work. We will review it and respond. 
+  
 ## Contributing translations
 
+#### Important Notice: We have paused all translation contributions. 
+> [!IMPORTANT]
+> We are currently working toward a more streamlined translations workflow. As long as this notice is posted, we will _not_ be accepting any translation submissions. 
```

Comment:
Considering that this will be reflected on the website, I wonder how it will look there, since it hasn't appeared yet. Maybe you should run the script to preview it.

Reply:
![image](https://github.com/user-attachments/assets/06c89d5e-d450-44d8-ba95-f690340eb882)

Reply:
Ahh your right.
I ran the script over and over on my account to test but it doesn't have the commit access to persist the file, so I must have missed this. Thanks!
Will figure out something here and get back.

Reply:
I've changed it to this in the website file in both sections it occurs. And checked the link at the top of the page that goes here still works.
![Screenshot 2024-11-18 at 9 27 46 PM](https://github.com/user-attachments/assets/57a9852c-f610-45e0-89ff-e39b0ad8276f)